docs/source/generic/rest.rst

```diff
@@ -119,7 +119,15 @@ Gebruik de :samp:`?_fields={veld1},{veld2},{...}` parameter om alleen specifieke
 
 .. code-block:: bash
 
-    curl 'https://api.data.amsterdam.nl/v1/bag/stadsdeel/?_fields=id,code,naam'
+    curl 'https://api.data.amsterdam.nl/v1/fietspaaltjes/fietspaaltjes/?fields=geometry,soortPaaltje'
+
+Als de veldnamen voorafgegaan worden door een minteken, dan worden alle velden behalve de genoemde
+opgestuurd:
+
+.. code-block:: bash
```

Comment:
Not sure if `bash` is correct. Sphinx delegates to Pygments and Pygments says: https://pygments.org/languages/#interactive-terminal-shell-sessions

Hence i presume it should either be:
```ReST
.. code-block:: console
```
or
```ReST
.. code-block:: shell-session
```

Reply:
Ah, never mind, `bash` is supported (as is `zsh` and `tcsh`)

Reply:
I double-checked and it looks fine in the generated HTML, with colored quoted strings and everything :)